### PR TITLE
Allows for in-app upgrade

### LINKED ISSUE #1631: ability to upgrade from within the app vs. cal.com/upgrade
### Is your proposal related to a problem?

right now cal.com/upgrade leads to a few problems, namely accounts sometimes are not associated with the stripe billing.

### Describe the solution you'd like

be able to upgrade/downgrade from within app.cal.com

### Describe alternatives you've considered
 
add an email parameter and prefill the email: cal.com/upgrade?email=peer@example.com

### PR DESCRIPTION
**Pull request by zomars**
## What does this PR do?

- Consolidates upgrade instructions so yo can upgrade via in-app
- Removes unused translations
- Upgrade instructions needs to be re-translated (hence the removal)

Fixes #1631

TODO: We should redirect all cal.com/upgrade links to app.cal.com/settings/billing on calendso/website

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [x] Upgrade the free account from event-types
- [x] Upgrade the free account from billing page
- [x] Upgrade the free account from "disable Cal.com branding" modal.
- [x] Upgrade the trial account from billing page
- [x] Upgrade the trial account from trial banner

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code and corrected any misspellings
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: public/static/locales/en/common.json [hunk 1]
```diff
@@ -451,9 +451,11 @@
   "url": "URL",
   "hidden": "Hidden",
   "readonly": "Readonly",
+  "plan_description": "You're currently on the {{plan}} plan.",
+  "plan_upgrade_invitation": "Upgrade your account to the pro plan to unlock all of the features we have to offer.",
   "plan_upgrade": "You need to upgrade your plan to have more than one active event type.",
   "plan_upgrade_teams": "You need to upgrade your plan to create a team.",
-  "plan_upgrade_instructions": "To upgrade, go to <a href=\"https://cal.com/upgrade\" className=\"underline\">https://cal.com/upgrade</a>",
+  "plan_upgrade_instructions": "You can <1>upgrade here</1>.",
```
Comment (emrysal) [inline public/static/locales/en/common.json:458]: Does this work?

Comment (zomars) [reply public/static/locales/en/common.json:458]: Yup. Html in there doesn't. But this does

Comment (jamiepine) [reply public/static/locales/en/common.json:458]: it doesn't seem to, clicking just redirects to the homepage

Comment (zomars) [reply public/static/locales/en/common.json:458]: You need to be running both calendso and the website. Since the upgrade logic is in there.

**Review by emrysal — APPROVED** [review]

**Review by PeerRich — APPROVED** [review]

**Pull request merged**